### DISCUSSION TITLE
Prisma with GraphQL/Rest DTO class?

### DISCUSSION THREAD

**Discussion by lkevenson** [prisma/prisma#7723] [Q&A] [answered]
Hello.
I try to implement a nestJS projects with Rest and GraphQL endpoint.
But I have issues with the Type from prisma and class on DTOs
How can i use prisma with some GraphQL/Rest DTO class?

**Comment by ryands17** [comment, marked as answer]
@lkevenson 👋 

There are some packages that you can check out that generate DTO and NestJS related stuff:

https://www.npmjs.com/package/@vegardit/prisma-generator-nestjs-dto
https://github.com/tpdewolf/prisma-nestjs-dto-generator

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
